src/gui/graphwidget.h

```diff
@@ -33,6 +33,7 @@ namespace evoplex {
 
 class BaseGraphGL;
 class GraphWidget;
+enum class SelectionMode;
```

Comment:
since the file already includes the `#include "basegraphgl.h"`, the lines 34-36 could be removed